src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month">
```

Comment:
```suggestion
    <div class="here should be classess which will modify length of calendar and start day of the month">
```

Reply:
@l4st1m0za 
![Знімок екрана 2024-01-13 о 16 43 34](https://github.com/mate-academy/layout_calendar/assets/152077319/00e5f7ec-4674-437a-9db2-af7ce069bda7)
I can't add multiple classes into attribute value because of lint rules)

Reply:
it should be modifiers for month.
something like that.
```suggestion
    <div class="month month--length-29 month--start-day-wed">
```
so add some modifiers for month block and fix styles for such modifiers.